libs/domain/cart/entries/src/entries.schema.ts

```diff
@@ -0,0 +1,12 @@
+import { ContentComponentSchema } from '@spryker-oryx/experience';
+import { cartEntryComponentSchema } from '../../entry/src/entry.schema';
```

Comment:
Can we use alias here?

Reply:
how?

Reply:
`@spryker-oryx/cart/entry`

Reply:
ah or we don't expose schema as public API, right?